extensions/amp-story-interactive/0.1/test/test-amp-story-interactive.js

```diff
@@ -31,6 +31,7 @@ import {LocalizationService} from '#service/localization';
 import {Services} from '#service';
 import {StoryAnalyticsService} from '../../../amp-story/1.0/story-analytics';
 import {dict} from '#core/types/object';
+import {expect} from 'chai';
```

Comment:
Nit: I think this import is unnecessary, can you check? (the file already used `expect` before this PR)

Reply:
I'll check. I didn't add that import myself, it must've been VSCode that added it automatically